vnext/Microsoft.ReactNative/ReactCoreInjection.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ReactCoreInjection.h"
+#include "ReactCoreInjection.g.cpp"
+#include "ReactViewOptions.g.cpp"
+#include "ReactInstanceWin.h"
+#include "ReactContext.h"
+#include "IReactContext.h"
+
+#include "ReactNativeHost.h"
+#include "DynamicWriter.h"
+
+namespace winrt::Microsoft::ReactNative::implementation {
+
+  ReactViewOptions::ReactViewOptions() noexcept {}
+
+  winrt::hstring ReactViewOptions::ComponentName() noexcept {
+    return m_componentName;
+  }
+
+  void ReactViewOptions::ComponentName(winrt::hstring value) noexcept {
+    m_componentName = value;
+      }
```

Comment:
Nit indentation

Reply:
Sigh.  At some point I need to look into fixing `yarn format` to work on new files.